gms/feature_service.cc

```diff
@@ -249,10 +249,8 @@ future<> feature_service::enable_features_on_startup(db::system_keyspace& sys_ks
         // enabled in the code by default, so just skip it.
     }
 
-    co_await container().invoke_on_all([&features_to_enable] (auto& srv) -> future<> {
```

Comment:
I don't get the patch title. It says "make it enable features on all shards", but current code _already_ _does_ _so_, doesn't it?

Reply:
The comment was about the "feature_service::enable" function. It didn't enable features on all shards _by itself_, and it had to be called via `invoke_on_all` to do that.

I did it mainly for consistency with the second, "peristent" overload of that function that I introduced in later commits.